test/pmi_client.c

```diff
@@ -334,30 +328,6 @@ static int test_item5(void)
 static int test_item6(void)
 {
     int rc = 0;
-    char val[100];
-    const char *tkey = __func__;
-    const char *tval = __FILE__;
-
-    if (PMI_SUCCESS != (rc = PMI_KVS_Put(jobid, tkey, tval))) {
-        log_fatal("PMI_KVS_Put %d\n", rc);
-        return rc;
-    }
-
-    if (PMI_SUCCESS != (rc = PMI_KVS_Get(jobid, tkey, val, sizeof(val)))) {
```

Comment:
PMI2 header is also not clear about this aspect:
https://github.com/jeffhammond/mpich/blob/master/src/include/pmi2.h#L286:L295